### PR TITLE
LPS-174905 Remove Stream usages for module: commerce-account-service

### PR DESCRIPTION
**Pull request by julschong**
resubmitted from #3588

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:sf

**Comment by julschong** [comment]
ci:test:relevant

**Comment by julschong** [comment]
ci:test:commerce

File: modules/apps/commerce/commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountGroupLocalServiceImpl.java [hunk 1]
```diff
@@ -249,16 +248,12 @@ public int compare(
 			return new ArrayList<>();
 		}
```
Comment (julschong) [inline modules/apps/commerce/commerce-account-service/src/main/java/com/liferay/commerce/account/service/impl/CommerceAccountGroupLocalServiceImpl.java:249]: Keeping the empty check

**Comment by Kvale1733** [comment]
@julschong No regressions found.

**Review by dantewang — APPROVED** [review]

**Comment by dantewang** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/130975

**Pull request closed without merging**